gradle.properties

```diff
@@ -0,0 +1,12 @@
+#Gradle
+org.gradle.jvmargs=-Xmx2048M -Dfile.encoding=UTF-8 -Dkotlin.daemon.jvm.options\="-Xmx2048M"
+
+#Kotlin
+kotlin.code.style=official
+
+#Android
+android.useAndroidX=true
+android.nonTransitiveRClass=true
+
+#MPP
+kotlin.mpp.enableCInteropCommonization=true
```

Comment:
Please add new line at the end of the file

Reply:
Updated at [f1f75f9](https://github.com/markgravity/kmm-ic/pull/50/commits/f1f75f96fa42793d71173dd2be60628857c4928a)